microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/jaxrs/java/JaxRsCodeLensParticipant.java

```diff
@@ -127,7 +135,9 @@ private static void collectURLCodeLenses(IJavaElement[] elements, String rootPat
 				if (isJaxRsRequestMethod(method) && Flags.isPublic(method.getFlags())) {
 					String baseURL = params.getLocalBaseURL();

```

Comment:
Replace with 

`String baseURL = context.getLocalBaseURL();`

and as I explained in https://github.com/eclipse/lsp4mp/pull/174#issuecomment-911410067 defines in JaxRsContext 

```java
public String getLocalBaseURL() {
   StringBuilder localBaseURL = new StringBuilder("http://localhost:");
  localBaseURL .append(getLocalServerPort()).toString();
   if (rootPath != null) {
      localBaseURL .append(getRootPath());
   }
  return localBaseURL.toString();
}
```